src/sfpowerkitCommand.ts

```diff
@@ -0,0 +1,71 @@
+import { SfdxCommand } from "@salesforce/command";
+import SFPKLogger, {COLOR_HEADER, LoggerLevel} from "./SFPKLogger";
+
+/**
+ * A base class that provides common funtionality for sfpowerscripts commands
+ *
+ * @extends SfdxCommand
+ */
+export default abstract class SFPowerkitCommand extends SfdxCommand {
+  public static isJsonFormatEnabled: boolean;
+  public static logLevel;
+  private sfpowerkitConfig;
+
+  /**
+   * Command run code goes here
+   */
+  abstract execute(): Promise<any>;
+
+  /**
+   * Entry point for the commands
+   */
+  async run(): Promise<any> {
+    // Always enable color by default
+    if (process.env.SFPOWERKIT_NOCOLOR) SFPKLogger.disableColor();
+    else SFPKLogger.enableColor();
```

Comment:
Could we use the existing logger `SFPowerkit` here, as I do not want to merge `SFPKLogger` unless it's being used throughout the codebase.

Reply:
removed